### ISSUE TITLE
训练不需要 prompt 引导

### ISSUE THREAD

**Issue by yqxd** [Francis-Rings/StableAvatar#58]
似乎发布的训练代码中都没有用 prompt 来做相关的控制，这是因为训练的时候只需要预测嘴部 mask 的部分来 in painting 吗，是否会造成 prompt 遵循能力的下降呢

**Comment by Francis-Rings** [comment]
Hi, the training code actually utilizes text prompt embeddings for conditioning.
```
with torch.no_grad():
                    prompt_ids = tokenizer(
                        batch['text_prompt'],
                        padding="max_length",
                        max_length=args.tokenizer_max_length,
                        truncation=True,
                        add_special_tokens=True,
                        return_tensors="pt"
                    )
                    text_input_ids = prompt_ids.input_ids
                    prompt_attention_mask = prompt_ids.attention_mask

                    seq_lens = prompt_attention_mask.gt(0).sum(dim=1).long()
                    prompt_embeds = text_encoder(text_input_ids.to(latents.device), attention_mask=prompt_attention_mask.to(latents.device))[0]
                    prompt_embeds = [u[:v] for u, v in zip(prompt_embeds, seq_lens)]
```

**Comment by Francis-Rings** [comment]
We simplify the text prompt and guide the diffusion model to learn the relationship between audio and synthesized facial expressions, which is crucial in audio-driven avatar video generation tasks. However, you can also use BLIP-2 or InternVL2 to generate text captions from your training videos and provide the corresponding text prompts to the dataloader, thereby enhancing the model’s ability to follow prompts.
